getting_started/step_by_step/ui_code_a_life_bar.rst

```diff
@@ -556,7 +556,7 @@ Try the game again to see a nice blocky animation.
 
 .. figure:: img/lifebar_tutorial_number_animation_working.gif
 
-   By rounding out animated\_health we hit two birds with one stone
+   By rounding out animated\_health, we killed two birds with one stone
```

Comment:
Maybe keep it in present tense: `kill`

Reply:
Also sounds better to me in present tense, I'll add that fixup.